aws-apprunner-nodejs-demo-app/README.md

```diff
@@ -6,15 +6,69 @@ The sample node application has been built on top of AWS AppRunner using typescr
 
 ## Installation
 
-TBD
+### Prerequisites
+
+You must have the [AppRunner](../aws-apprunner) installed already and the details of the application URL and the authentication token available.
+This application uses the Vault typescript SDK version 1.0.8.
+
+## Steps
+
+1. Clone this repository
+1. Move this directory into a new repository of its own. Let's call it `demo-app` under your account `myaccount`. It is assumed that the repository will be hosted in: `https://github.com/myaccount/demo-app`
+1. Modify [apprunner.yaml](apprunner.yaml):
+   1. Change `PVAULT_URL` containing generic xxxx. to the right URL of the Vault
+   1. Change `PVAULT_API_KEY` account number from the generic `123456789012` into your account number
+1. Open the AWS Console in the AppRunner section: https://us-east-2.console.aws.amazon.com/apprunner
+1. Click "Create service" and follow these steps.
+
+   1. Choose Source code repository
+
+      ![Step1](./resources/AppRunner-step1-auto.png)
+
+   1. Connect to Github and replace repository name with your own repository
+
+      ![step1b](./resources/AppRunner-step1-source.png)
+
+   1. Instead of Piiano, you should have your own application (click Install) and it will pop up a connection to GitHub. Select your organization or yourself
+
+      ![step1c](./resources/AppRunner-connect-to-github.png)
+
+   1. Select automatic mode as the `Deployment settings`
+
+1. Select to use a [configuration file](apprunner.yaml) in the `build settings`.
+
+   ![Step2](./resources/AppRunner-step2.png)
+
+1. Settings for step 3: `Configure service`.  
+   Don't forget to **set the same VPC as the Vault** in the networking section.
+
+   ![Step3](./resources/AppRunner-step3.png)
+
+1. Setting for step 3: `Health check`
+
+   ![Step3](./resources/AppRunner-step3-health.png)
```

Comment:
Did you test it? Does it really need port 3000?
I think it's a mistake here and in the other curl commands below